action.yaml

```
@@ -21,9 +21,13 @@ inputs:
     default: 'pytest'
     required: false
   package_name:
-    description: 'Name of package on PyPi, only needed if testing package post-upload'
+    description: 'Name of package on PyPi, only needed if testing package post-upload. Used for pip installing'
     default: 'MDAnalysis'
     required: false
+  module_name:
+    description: 'Name of package directory, only needed if testing package post-upload. Used for pytest pyargs'
```

```suggestion
    description: 'Name of package directory, only needed if testing package post-upload. Used for pytest pyargs. If unset will use `package_name`'
```
Something like that?

---

Or we could make it an explicit requirement.

---

Up to you! I think this is a relatively uncommon scenario though, right?

---

Extra documentation never hurt anyone 😉